frontend/src/utils/customHooks/useDisableElementsOnSomeRoutes.js

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+
+export const useDisableElementsOnSomeRoutes = props => {
```

Comment:
I think we should change the name of this hook. 
What we do here really, is just checking if the user is on the certain route or routes.

Reply:
Changed we were worked together